terraform/ratingservice/main.tf

```diff
@@ -91,7 +93,7 @@ locals {
 }
 
 resource "google_storage_bucket" "it" {
-  name                        = "${var.gcp_project_id}-ratingservice-deployables-${random_id.suffix.hex}"
+  name                        = "${var.gcp_project_id}-ratingservice-${random_string.suffix.result}"
```

Comment:
I wasn't sure if this would fit, so I did the math:
- max project id = [30 characters](https://cloud.google.com/resource-manager/docs/creating-managing-projects)
- max bucket name = [63 characters](https://cloud.google.com/storage/docs/naming-buckets#:~:text=Bucket%20names%20must%20start%20and,no%20longer%20than%2063%20characters.)

30 + 15 + 4 < 63, so this works. You may want to add a comment explaining the limit so we don't increase this too much in the future

Reply:
I will do.